fleet-commander.spec

```diff
@@ -5,8 +5,9 @@ Summary:        Fleet Commander
 
 #BuildRequires:  python3-gobject
 BuildRequires:  python
-BuildRequires:  python-requests
 BuildRequires:  dconf
+BuildRequires:  libvirt-python
+BuildRequires:  dbus-python
```

Comment:
We're prolly missing pygobject2 here. Would be worth our while figuring out how we can migrate the script towards pygobject3 as pygobject2 is pretty much unmaintained.